encrypted-media-respec.html

```diff
@@ -579,15 +579,14 @@
 
         <dt id="time">Time</dt>
         <dd>
-          <p>Time represents an instant in time with millisecond accuracy.  The instants in time it can represent are the same that can be represented with 
-            <a class="external" href="https://tc39.github.io/ecma262/#sec-date-objects">ECMAScript <span class="estype">Date</span> objects</a> (ECMA-262, section 20.3 [[!ECMA-262]]) – 
-            namely, every millisecond in the 200,000,000 days centered around midnight of 1 January, 1970 UTC, except for any millisecond that is part of an inserted leap second.  Time does not include these added leap milliseconds.
+          <p>Time MUST be equivalent to that represented in <a class="external" href="https://tc39.github.io/ecma262/#sec-time-values-and-time-range">ECMAScript <span class="estype">Time Values and Time Range</span></a> (ECMA-262, section 20.3.1.1 [[!ECMA-262]]). 
```

Comment:
I worry that the section number will rot, especially since the URL target is a living document. Also, we don't call out sections or even the name of documents elsewhere (except by ``[[foo]]``). Is there a reason you did it this way?